WakeUpAlarm/Models/Alarm.swift

```diff
@@ -14,19 +14,41 @@ import Foundation
 
```

Comment:
Really like the merged class. Left a few comments below - mostly nits / variable naming

Reply:
To avoid crowding with too many comments, not commenting where all the changed names would have to be updated in other files